lib/plan-builder.js

```diff
@@ -84,6 +84,9 @@ class Builder extends bldrgen.PlanBuilder {
     if(this.rdt === void 0) {
       this.rdt = new RdtExpr();
     }
+    if(this.ctsString === void 0){
+      this.ctsString = new CtsExpr();
```

Comment:
I think we need to take the same approach in the Java Client and modified the generated `CtsExpr` class in plan-builder-generated.js. While that's not desirable, I think it's better to provide a simple, consistent UX for a developer rather than to complicate the UX by introducing `ctsString` - we're making things more complicated for the user solely because we have a buggy code generator. Until the code generator is fixed, the right solution is to modify the buggy generated code.